File: 2482.cpp
struct Solution {
    vector<vector<int>> onesMinusZeros(vector<vector<int>>& grid) {
        int m = grid.size(), n = grid[0].size();
        vector<int> oneRow(m), oneCol(n);
        for (int i = 0; i < m; ++i){
            for (int j = 0; j < n; ++j){
                oneRow[i] += grid[i][j];
                oneCol[j] += grid[i][j];
            }
        }
        for (int i = 0; i < m; ++i){
            for (int j = 0; j < n; ++j){
                grid[i][j] = (oneRow[i] + oneCol[j]) * 2 - m - n;
            }
        }
        return grid;
    }
};

File: 2482.py
class Solution:
    def onesMinusZeros(self, grid: List[List[int]]) -> List[List[int]]:
        m, n = len(grid), len(grid[0])
        oneRow = list(0 for _ in range(m))
        oneCol = list(0 for _ in range(n))
        for i in range(m):
            for j in range(n):
                if grid[i][j]:
                    oneRow[i] += 1
                    oneCol[j] += 1
        for i in range(m):
            for j in range(n):
                grid[i][j] = (oneRow[i] + oneCol[j]) * 2 - m - n
        return grid
